models/_sector/alm/ethereum/alm_ethereum_base_trades.sql

```diff
@@ -0,0 +1,63 @@
+{{ config(
+    schema = 'alm_ethereum'
+    , alias = 'base_trades'
+    , materialized = 'view'
+    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'vault_address']
```

Comment:
Since this spell is materialized as `view` now, I will remove incremental filters and unique_key in this spell.